src/chain/node.js

```diff
@@ -103,6 +103,9 @@ async function getBalance (address, { height, hash, format = AE_AMOUNT_FORMATS.A
   return formatAmount(balance, { targetDenomination: format }).toString()
 }
 
+/**
+ * @deprecated use `sdk.api.getTransactionByHash/getTransactionInfoByHash` instead
+ */
 async function tx (hash, info = true) {
   const tx = await this.api.getTransactionByHash(hash)
   if (['ContractCreateTx', 'ContractCallTx', 'ChannelForceProgressTx'].includes(tx.tx.type) && info && tx.blockHeight !== -1) {
```

Comment:
For these transactions types, it would make two requests to the node even if txInfo is not needed. Also, we have additional transaction types that have non-empty transaction info, seems irrational to add them here, so proposing to use plain endpoint instead.